docs/application-connector/docs/005-architecture-application-connector.md

```diff
@@ -5,11 +5,72 @@ type: Architecture
 
 The Application Connector consists of the following components:
 
-* **Ingress-Gateway controller** responsible for validating certificates and exposing multiple Application Connectors to the external world
-* **Gateway** responsible for registering available services (APIs, Events) and proxying calls to the registered solution
-* **Remote Environment CRD instance** responsible for storing a solution's metadata
-* **Minio bucket** responsible for storing API specifications, Event Catalog, and documentation
+* **Nginx Ingress Controller**
+* **Connector Service** 
+* **Metadata Service**
+* **Event Service**
+* **Proxy Service**
+* **Remote Environments**
+* **Minio bucket**
+* **Kubernetes Secrets**
+* **Kubernetes Services**
 
-To connect a new external solution, you must install and set up a new Remote Environment. Every external solution connected to Kyma is a separate Remote Environment with a dedicated Gateway Service and a dedicated Event Service. See the **Set up a Remote Environment on local Kyma installation** getting started guide to learn how to connect an external solution to Kyma.
 
 ![Architecture Diagram](assets/001-application-connector.png)
+
+
+### Nginx Ingress Controller
```

Comment:
don't do subsections here, you have separate docs for more details about each component, here focus on describing the diagram and architecture in overall